tsconfig.app.json

```diff
@@ -0,0 +1,15 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
```

Comment:
Why the need for the extra tsconfig file?

Reply:
We don't, just removed them